inference-engine/src/vpu/graph_transformer/src/frontend/frontend.cpp

```diff
@@ -235,7 +235,140 @@ ie::CNNNetwork FrontEnd::convertNetwork(ie::CNNNetwork& network) {
     IE_SUPPRESS_DEPRECATED_END
 }
 
-std::set<std::string> FrontEnd::checkSupportedLayers(const ie::CNNNetwork& network) {
+std::vector<ie::CNNNetwork> FrontEnd::checkSupportedNetworks(const ie::CNNNetwork& network, std::set<std::string>& namesToExclude) {
+    std::vector<ie::CNNNetwork> supportedNetworks;
+    if (network.getFunction() && network.getFunction()->is_dynamic()) {
+        auto copyNetwork = InferenceEngine::details::cloneNetwork(network);
+        try {
+            convertNetwork(copyNetwork);
+        } catch(vpu::details::VPUException e) {
+            std::string name = e.what();
+            std::string nameDelimiter("node ");
+            name.erase(0, name.find(nameDelimiter) + nameDelimiter.length());
+            nameDelimiter = " of";
+            name.erase(name.find(nameDelimiter), name.length());
```

Comment:
I don't quite understand it too :) Are we looking for the node name from the error message? If so, I think it's worth at least specifying the format of the error message we expect to see (by comment), so that if this message changes for some reason, we could update this part of the algorithm.

Reply:
I'll add a comment with what error massage should look like for this part to work.

Reply:
I've added it.